### ISSUE TITLE
ERROR: INVALID CLIENT

### ISSUE THREAD

**Issue by basicgodly** [bddvlpr/bad-vrc-osc#1]
Once clicking the localhost:8888/callback the application crashes prompting a Invalid Client Error
Tried to analyze/fix this error with no success.

I might have a idea why, which points out to one of the steps saying:
"Copy **.env.example** to **.env** and fill in the _'Client ID'_ and _'Client Secret'_ you obtained above". The file stays as EXAMPLE extension and I've tried making a new TXT with the inside contents, but with no result.
Is there a specific extension I need to save this said file in?
![image_2022-08-30_201027873](https://user-images.githubusercontent.com/46283142/187511485-1552b708-db3d-4c98-89f7-4e13a83e6cb8.png)
![image_2022-08-30_201509160](https://user-images.githubusercontent.com/46283142/187512353-ce95fe8f-7484-4256-a967-dae6a43f32ea.png).

**Comment by bddvlpr** [comment]
Your file is called `.env.txt`, not `.env` (as seen by the ‘Text Document type’). Enable ‘Show file extensions’ in the Windows Explorer and rename it correctly.

> Click the “View” tab along the top of the File Explorer window. Mouse over “Show” at the bottom of the drop-down menu, and then click “File Name Extensions” in the sub-menu.
*Source: https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/amp/*

**Comment by basicgodly** [comment]
I totally forgot that was a thing, that has fixed the invalid_client error, but now we got a different error.
I recloned the Git, renamed the file to .env and I put my CLIENT ID and SECRET KEY in the .env and now I get hit by invalid_request. 
Double checked everything, reset the secret key and replaced it, even tried recloning again.
![image_2022-08-31_201155547](https://user-images.githubusercontent.com/46283142/187749647-88c189ab-818a-40de-8866-8aef79135b6f.png)
![image_2022-08-31_201627156](https://user-images.githubusercontent.com/46283142/187750431-fa9c4334-1a6c-437f-96f5-301e06bcae37.png)
![image_2022-08-31_202018868](https://user-images.githubusercontent.com/46283142/187751157-dadafdb9-d8c4-4546-8f5f-2f1706972bc8.png)
![image_2022-08-31_201638465](https://user-images.githubusercontent.com/46283142/187750468-a974e615-b5a9-4a95-b62b-785953a7c96f.png)
